Sources/Sentry/SentryScope.m

```diff
@@ -583,18 +583,6 @@ - (SentryEvent *__nullable)applyToEvent:(SentryEvent *)event
         event.level = level;
     }
 
-    NSMutableDictionary *newContext = [self context].mutableCopy;
-    if (event.context != nil) {
-        [SentryDictionary mergeEntriesFromDictionary:event.context intoDictionary:newContext];
-    }
-
-    // Don't add the trace context of a current trace to a crash event because crash events are from
-    // a previous run.
-    if (event.isCrashEvent) {
-        event.context = newContext;
-        return event;
-    }
```

Comment:
We don't need this anymore because we don't call this for crash events anymore in the client.

Reply:
How can we assure it won't be called for crashes again in the future?

Reply:
I will add an early exit at the top of the method.